src/server.c

```diff
@@ -694,7 +694,7 @@ int allPersistenceDisabled(void) {
 /* ======================= Cron: called every 100 ms ======================== */
 
 /* Add a sample to the operations per second array of samples. */
-void trackInstantaneousMetric(int metric, long long current_reading) {
+void trackInstantaneousRateMetric(int metric, long long current_reading) {
     long long now = mstime();
```

Comment:
maybe we can move the `mstime()` call to outside? (it's silly that it is called inside a loop)